packages/shared/src/components/cards/SharePostCard.tsx

```diff
@@ -51,15 +51,17 @@ export const SharePostCard = forwardRef(function SharePostCard(
   const isVideoType = isVideoPost(post);
   const improvedSharedPostCard = useFeature(feature.improvedSharedPostCard);
 
-  // Just for this experiment, we are modifying the post object to use set the
-  // image from the shared post, this is just to avoid modifying the `PostCardFooter`
-  // component to handle this case just for this experiment.
-  if (improvedSharedPostCard) {
-    // eslint-disable-next-line no-param-reassign
-    post.image = post.sharedPost.image;
-    // eslint-disable-next-line no-param-reassign
-    post.sharedPost.createdAt = post.createdAt;
```

Comment:
createdAt isn't used in PostCardFooter, and I've got a fix in #3224 that removes the param reassign for it.

Reply:
This is more to prevent param reassign, but yeah I'll remove `createdAt` if not needed or when yours is merged.